dist/stat_data.py

```diff
@@ -244,6 +244,8 @@ def __init__(self, name, desc, flags=''):
     BlockCacheStat('block_prefetch_pages_queued', 'pre-fetch pages queued'),
     BlockCacheStat('block_prefetch_pages_read', 'pre-fetch pages read in background'),
     BlockCacheStat('block_prefetch_skipped', 'pre-fetch not triggered by page read'),
+    BlockCacheStat('block_prefetch_skipped_no_leaf_page', 'pre-fetch not triggered on internal page, pre-fetch should only see leaf pages'),
```

Comment:
Not a strong opinion, but I'd rather have this as something like `block_prefetch_skipped_internal_page`.